mindcv/optim/optim_factory.py

```diff
@@ -59,19 +219,52 @@ def create_optimizer(
             of current step. Default: 0.
         momentum: momentum if the optimizer supports. Default: 0.9.
         nesterov: Whether to use Nesterov Accelerated Gradient (NAG) algorithm to update the gradients. Default: False.
-        filter_bias_and_bn: whether to filter batch norm parameters and bias from weight decay.
-            If True, weight decay will not apply on BN parameters and bias in Conv or Dense layers. Default: True.
+        weight_decay_filter: filters to filter parameters from weight_decay.
+            - "disable": No parameters to filter.
+            - "auto": We do not apply weight decay filtering to any parameters. However, MindSpore currently
+                    automatically filters the parameters of Norm layer from weight decay.
```

Comment:
Be specific. For 'auto' weight decay, `Norm layer` , which kind of norm layers will be applied with weight decay? BatchNorm only or including LayerNorm

Reply:
All the norm layers BatchNorm and LayerNorm.